tests/test_playbooks/vars/hostgroup.yml

```diff
@@ -27,6 +29,13 @@ hostgroup:
   subne6: Test subnet6
   puppet_server: "{{ foreman_proxy }}"
   puppet_ca: "{{ foreman_proxy }}"
+  dns_proxy: "{{ foreman_proxy }}"
+  openscap_proxy: "{{ foreman_proxy }}"
+  content_source: "{{ foreman_proxy }}"
```

Comment:
I am wondering, this does not collide with the plain-foreman version of the playbook.

Reply:
I don't think, this is doted notation, and we use vars name with underscores for playbooks/tasks variables so for me no collisions.

Reply:
Sorry, i mean the fact, that you provide `content_source` which is a katello thing in the foreman only test. But i guess, in the end, it is silently filtered by apypie.

Reply:
Yes, I already ran hostgroup tests after adding that and all went fine.